components/header-bar/src/header-bar.stories.e2e.js

```diff
@@ -23,3 +23,15 @@ export const PWAEnabled = () => (
         <HeaderBar appName="Example!" />
     </Provider>
 )
+
+export const WithLastOnline = () => (
+    <Provider
+        config={{
+            ...config,
+            pwaEnabled: true,
+            headerbar: { onlineStatusInfo: 'last-online' },
```

Comment:
Using uppercase snake case (`LAST_ONLINE`) would make it clearer that this value is an enum and won't be rendered directly.